app/views/miq_request/_st_prov_show.html.haml

```diff
@@ -11,22 +11,11 @@
       .row
         .col-md-12.col-lg-12
           #dialog_tabs
-            %ul.nav.nav-tabs{'role' => 'tablist'}
+            - catalog_tab_labels = st_prov_show_tab_configuration(wf)            
+            = react('MiqCustomTab', {:containerId => 'st-prov-show-tabs-content',
+                          :tabLabels => catalog_tab_labels,
+                          :type => 'CATALOG_SUMMARY'})
```

Comment:
I think you need to add a new type here. maybe `PROV_DETAILS` will do. This is to identify where these tabs are used.